packages/matter.js/src/cluster/client/EventClient.ts

```diff
@@ -19,37 +20,40 @@ export class EventClient<T> {
         private getInteractionClientCallback: () => Promise<InteractionClient>,
     ) { }
 
-    async get(_alwaysRequestFromRemote = false) {
+    async get(minimumEventNumber?: number | bigint, isFabricFiltered?: boolean): Promise<DecodedEventData<T>[] | undefined> {
```

Comment:
I'm curious, why is `isFabricFiltered` a parameter on get/subscribe rather than an property of `EventClient`?

Reply:
The controller defines if he want to reasd te value fabricFiltered or "all" (not fabric filtered), so it is a parameter of the get/subscribe request and the response will differ potentially